Gemfile

```diff
@@ -65,6 +65,7 @@ group :test, :development do
   gem 'rack-test', :require => 'rack/test'
   gem 'simplecov', :require => false
   gem 'pry-byebug'
+  gem 'pry-doc'
```

Comment:
How does it relate to the rest of the commit? If I was coming across this commit later, it may be a red herring to see this gem change with the rest of the changes. That's why I would put this with another PR, unless it has direct relevance to the commit?